src/views/CreateInstance.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <main v-if="formState === 0" class="create-Instance">
+    <ViewHeader
+      :title="`Choose a resource type for the new instance`"
+      :backLink="{ link: '/resources' }"
+    />
+    <ListSection :list="nonAbstractResourceTypesList"/>
+  </main>
+
+  <main v-else-if="formState === 1" class="create-Instance">
+    <ViewHeader title="Choose attribute values for the new Instance" :backLink="{ onClick: previousStep }"/>
+    <ListSection :title="`Attributes of ${resourceType.name}`">
+      <Li v-for="attribute in newResourceInstance.attributes" :key="attribute.name">
+        <TypeSensitiveInput
+          :resourceType="newResourceInstance.resourceType"
+          :resourceInstanceAttribute="attribute"
+        />
+      </Li>
+    </ListSection>
+
+    <Button text="save" :onClick="createResourceInstance"/>
+  </main>
+</template>
+
+<script lang="ts">
+import { Component, Mixins } from 'vue-property-decorator';
+import {
+  ResourceTypes,
+  ResourceType,
+  ResourceTypeAttribute,
+  ResourceInstances,
+} from '@/apis/rembrandt/rembrandt';
+import Li, { ListEntry } from '@/components/Li.vue';
+import ViewHeader from '@/components/ViewHeader.vue';
+import ListSection from '@/components/ListSection.vue';
+import TypeSensitiveInput from '@/components/TypeSensitiveInput.vue';
+import Input from '@/components/Input.vue';
+import Toggle from '@/components/Toggle.vue';
+import Button from '@/components/Button.vue';
+import Select from '@/components/Select.vue';
+import Utils from '@/utils/Utils';
+import Translate from '@/mixins/Translate';
+import { ResourceInstance, ResourceInstanceNullObject } from '@/apis/rembrandt/rembrandt';
+
+@Component({
+  components: {
+    ListSection,
+    Li,
+    ViewHeader,
+    TypeSensitiveInput,
+    Input,
+    Toggle,
+    Button,
+    Select,
+  },
+})
+export default class Types extends Mixins(Translate) {
+  // region public static methods
+  public static emptyResourceType(): ResourceType {
```

Comment:
Why not use the `ResourceTypeNullObject` here?
https://github.com/bptlab/rembrandt-frontend/blob/df1ea54fde75620c5680b8ec4db652f39570fcc7/src/apis/rembrandt/lib/ResourceType.ts#L37

Reply:
changed it